operator/internal/manifests/config.go

```diff
@@ -102,6 +103,12 @@ func ConfigOptions(opt Options) config.Options {
 		}
 	}
 
+	gossippRingconfig := gossipRingConfig(opt.Name, opt.Namespace, opt.Stack.HashRing)
+
+	if zoneAwareEnabled {
+		gossippRingconfig.Zone = zoneAwareConfig(opt.Stack.Replication)
+	}
```

Comment:
Sure, I did consider it. The reason I kept it separate is, I thought adding Zone related configs directly into gossipRing, might be confusing (even though that's how it is finally added in the loki-config.yaml). Also a different struct would be good, if at some point in the future additional zone related configs might get added. But basically I am open to blending it into the gossipring struct too since a different struct is not an immediate requirement.

Reply:
IMHO future additional zone configs are a thing of the future, thus we can expand to a separate struct if this future materializes somehow.